mappings/servers/spalion/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "spalion",
+    "name": "Spalion",
+    "description": "French Ranked Game Like BedWars",
+    "website": "https://spalion.net/",
+    "addresses": [
+        "spalion.net",
+        "mc.spalion.net",
+        "play.spalion.net",
+        "spalion.fr",
+        "mc.spalion.fr",
+        "play.spalion.fr",

```

Comment:
As stated by the checklist. Don't include subdomains here. They are already covered.